packages/renovate-config/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@vtex/renovate-config",
+  "version": "0.368.1",
+  "description": "A plugin that holds the Renovate preset config for updating VTEX dependencies in our stores",
+  "author": "VTEX",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vtex/faststore.git"
+  },
+  "renovate-config": {
+    "default": {
+      "extends": [
+        "config:base",
+        ":timezone(America/Sao_Paulo)"
+      ],
+      "labels": [
+        "renovate",
+        "automerge",
+        "VTEX"
+      ],
+      "schedule": [
+        "after 9am and before 6pm every weekday"
+      ],
+      "reviewers": [
+        "vtex:faststore",
+        "vtex:store-framework"
+      ],
+      "pin": {
+        "automerge": true
```

Comment:
Are we doing automerge?

Reply:
I want to test the automerge behavior when our pipelines fails, but gonna change after those tests.